### PR TITLE
[DO-1777] check Storing property and Storing luxuries from Declaration

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

**Review by roman-tiukh — APPROVED** [review]

File: business_register/converter/declaration.py [hunk 1]
```diff
@@ -840,12 +851,18 @@ def save_spouse(self, relatives_data, pep, declaration):
...
             to_person_relationship_type = relative_data.get('subjectRelation')
             if to_person_relationship_type in SPOUSE_TYPES:
                 spouse = None
                 nacp_id = relative_data.get('id')
                 if nacp_id:
-                    spouse = Pep.objects.filter(nacp_id=nacp_id).first()
+                    if nacp_id == 'declarationType':
+                        self.log_error(f'Invalid value: nacp_id = {nacp_id}')
+                    else:
+                        spouse = Pep.objects.filter(nacp_id=nacp_id).first()
```
Comment (Iva-khar) [inline business_register/converter/declaration.py:865]:
```suggestion
                    if isinstance(nacp_id, int):
                        spouse = Pep.objects.filter(nacp_id=nacp_id).first()
                    else:
                        self.log_error(f'Invalid value: nacp_id = {nacp_id}')
```

Comment (Iva-khar) [reply business_register/converter/declaration.py:865]: Maybe...

File: business_register/converter/declaration.py [hunk 2]
```diff
@@ -840,12 +851,18 @@ def save_spouse(self, relatives_data, pep, declaration):
         SPOUSE_TYPES = ['дружина', 'чоловік']
         # TODO: decide should we store new Pep that not spouse from relatives_data
         for relative_data in relatives_data:
+            if type(relative_data) == str:
+                self.log_error(f'Invalid value: relative_data = {relative_data}')
+                break
```
Comment (Iva-khar) [inline business_register/converter/declaration.py:856]: What if `relative_data` is `int `or `float` for example?

**Review by Iva-khar — APPROVED** [review]

File: business_register/converter/declaration.py (hunk 1 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py:865]: I saw the wrong id value only nacp_id = 'declarationType' and in some declaration nacp_id = 0

File: business_register/converter/declaration.py (hunk 2 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py:856]: I have not seen int and float values

Comment (Iva-khar) [reply business_register/converter/declaration.py:856]: Must check that type what kind you are waiting for. The same in previous comment.

File: business_register/converter/declaration.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/converter/declaration.py:865]: When will appear new wrong type you will rewrite it?

File: business_register/converter/declaration.py [hunk 3]
```diff
@@ -47,7 +47,8 @@ def __init__(self):
             '[Не застосовується]',
             '[Не відомо]',
             "[Член сім'ї не надав інформацію]",
-            '[Конфіденційна інформація]'
+            '[Конфіденційна інформація]',
+            'Не визначено'
```
Comment (Iva-khar) [inline business_register/converter/declaration.py:51]: It seems strange. So different formatting...

**Review by Iva-khar — CHANGES_REQUESTED** [review]

File: business_register/converter/declaration.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/converter/declaration.py:865]: & `nacp_id = 'declarationType' ` it's string, not object of declaration isn't it?

File: business_register/models/declaration_models.py [hunk 4]
```diff
@@ -759,7 +759,7 @@ class Property(DataOceanModel):
         verbose_name=_('address'),
         help_text=_('city where the property is located')
     )
-    valuation = models.PositiveIntegerField(
+    valuation = models.FloatField(
```
Comment (criticus) [inline business_register/models/declaration_models.py:762]: float or decimal?

File: business_register/converter/declaration.py (hunk 2 above)
Comment (roman-tiukh) [reply business_register/converter/declaration.py:856]: @Iva-khar +++

File: business_register/models/declaration_models.py (hunk 4 above)
Comment (roman-tiukh) [reply business_register/models/declaration_models.py:762]: if this is money then should be decimal

File: business_register/converter/declaration.py (hunk 1 above)
Comment (roman-tiukh) [reply business_register/converter/declaration.py:865]: @Iva-khar +

**Review by roman-tiukh — APPROVED** [review]

**Review by Iva-khar — APPROVED** [review]

File: business_register/converter/declaration.py (hunk 1 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py]: fixed

File: business_register/converter/declaration.py (hunk 2 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py:858]: fixed

File: business_register/models/declaration_models.py (hunk 4 above)
Comment (EmmaSamoilova) [reply business_register/models/declaration_models.py]: done

**Pull request merged**